src/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RootContainer from './RootContainer';
+
+// make sure to export main, with the signature
+function main(el, service, imEntity, state, config) {
+	if (!state) state = {};
+	if (!el || !service || !imEntity || !state || !config) {
+		throw new Error('Call main with correct signature');
+	}
+	ReactDOM.render(
+		<RootContainer
+			serviceUrl={service.root}
+			entity={imEntity.Gene}
```

Comment:
*config.json* specifies `"classes": ["Gene", "Protein"]` so we need to handle the case of a protein class. I would recommend simply changing to `entity={imEntity.Gene || imEntity.Protein}`.